client/package.json

```diff
@@ -8,6 +8,7 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
+    "postinstall": "ngcc",
```

Comment:
I think this actually slows down our CI builds because with this I think it has to generate all possible package formats for all libraries in `node_modules` whereas `ng serve` and `ng build` will run `ngcc` as needed automatically.

Reply:
I suppose that can be mitigated if we have GH Actions cache the compiled modules.

Reply:
Do you think we should just remove this altogether? I haven't looked into this in any details; what they say in the upgrade guide is

> If you depend on many Angular libraries you may consider speeding up your build by invoking the ngcc (Angular Compatibility Compiler) in an npm postinstall script via small change to your package.json.

I wasn't clear on what constituted "many Angular libraries", but it seemed like we depended on quite a few so I thought this might improve performance for students. Are you saying that `ng serve` will automagically call `ngcc` so we don't need this in the `package.json`?

Reply:
After updating Material and rxjs I get this warning:

> Warning: The inferred tsconfig file "/Users/mcphee/Downloads/3601-iteration-template/client/tsconfig.json" appears to be "solution-style" since it contains no root files but does contain project references.
This is probably not wanted, since ngcc is unable to infer settings like "paths" mappings from such a file.
Perhaps you should have explicitly specified one of the referenced projects using the --tsconfig option. For example:

>  ngcc ... --tsconfig "./tsconfig.app.json"
>  ngcc ... --tsconfig "./tsconfig.spec.json"

> Find out more about solution-style tsconfig at https://devblogs.microsoft.com/typescript/announcing-typescript-3-9/#solution-style-tsconfig.
> If you did intend to use this file, then you can hide this warning by providing it explicitly:

>  ngcc ... --tsconfig "tsconfig.json"
> removed 1 package and audited 1631 packages in 10.374s

I didn't get this on the other project, so I'm not really sure what triggered it. Everything still seems to work (e.g., tests pass), but this does suggest that best we don't have things set up "best" for `ngcc`.

Should I just remove this line?